api/src/services/items.test.ts

```diff
@@ -56,6 +57,30 @@ describe('Integration Tests', () => {
 
 				expect(validateUserCountIntegrity).toHaveBeenCalled();
 			});
+
+			it('should detect MS SQL client and prepare correct returning options', () => {
+				// Mock MS SQL client
+				vi.mocked(getDatabaseClient).mockReturnValueOnce('mssql');
+
+				// Test the logic that determines returning options
+				const client = getDatabaseClient();
+				const returningOptions = client === 'mssql' ? { includeTriggerModifications: true } : undefined;
```

Comment:
Lets revise this so we don't repeat the actual business logic in the test. We don't need the `client === 'mssql'` branching here as we've already mocked `getDatabaseClient` to output `mssql`